frontend/src/modules/history/components/Translation.css

```diff
@@ -18,7 +18,11 @@
 
 .history .translation .avatar img {
     border-radius: 4px;
-    border: 1px solid #4D5967;
+    border: 1px solid #5E6475;
+}
+
+.history .translation .avatar img:hover {
```

Comment:
We should change the border color on `.translation:hover` (when the background changes), not `.img:hover`.